### ISSUE TITLE
Allow naming of plots/tables

### ISSUE THREAD

**Issue by lazappi** [csoneson/iSEE#125]
Wondering if it would be possible to allow users to name each plot/table? I think this would help keep track of which things are related to each other and communication with other people. It would also be handy if you could specify this in the defaults so when you open it it's obvious what the plots are supposed to show.

P.S. This looks like it will be an awesome tool, thanks for putting in the hard work to put it together!

**Comment by LTLA** [comment]
Hm. I have thought about this, and that answer is - not easily. If you change the names, you need to update the `selectInput` choices that link between plots and/or tables. In fact, you'd need to re-render the UI completely every time there's a name change, as the title of each panel lies within `shinydashboard::box`; there doesn't seem to be an easy way to use `uiOutput` to just update the title.

In short, it's not impossible, but it's not easy, either. 

In the meantime, we should probably put some more information in the plot title to indicate what it's showing, as sometimes it's not obvious (e.g., reduced dimension plots could be anything, really).

**Comment by LTLA** [comment]
Naming of reduced dimensions has been implemented in #131. Probably still needs a little bit of work to harden it against unnamed reduced dimensions, but otherwise it should suffice.

Alternatively, we could also put in sensible automatic titles in the _ggplot_ itself. This is probably faster for users to read, and should relieve the burden on the axes to explain everything. 

To sum up: at this point, we are not going to allow users to change the names of the panels. This would break some parts of the current system of how plots are linked together; it would complicate how users can specify links between plots in the default options; and it would require re-rendering of the entire UI for any single name change. I don't think it's worth it, especially if the automatic titles do a good job.

**Comment by kevinrue** [comment]
Fixed by #131
